src/Microsoft.Health.Fhir.Core/Features/Operations/Export/Models/ExportJobProgress.cs

```diff
@@ -27,12 +29,30 @@ protected ExportJobProgress()
         [JsonProperty(JobRecordProperties.Page)]
         public uint Page { get; private set; }
 
+        [JsonProperty(JobRecordProperties.SubSearchResourceId)]
+        public string SubSearchResourceId { get; private set; }
+
+        [JsonProperty(JobRecordProperties.SubSearch)]
+        public ExportJobProgress SubSearch { get; private set; }
+
         public void UpdateContinuationToken(string continuationToken)
         {
             EnsureArg.IsNotNullOrWhiteSpace(continuationToken, nameof(continuationToken));
 
             ContinuationToken = continuationToken;
             Page++;
         }
+
+        public void NewSubSearch(string resourceId, ExportJobProgress subSearch)
```

Comment:
No callers?

Reply:
None yet. This story was meant to brainstorm ideas on how to cover the use case described above. The change was small enough that instead of a design doc I decided to just implement the change, and review it with this PR. I'll implement its use in the next story: User Story 74398: Add Compartment resources to Patient export